src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksNewVendorRequestsServiceImpl.java

```diff
@@ -281,6 +278,16 @@ private boolean isoCountryCodeTranslatesToSingleFipsCountryCode(String isoCountr
         }
     }
     
+    private boolean bankIsoCountryCodeTranslatesToSingleFipsCountryCodeWhenBankDataIsEntered(String isoCountryCode, List<String> errorMessages) {
```

Comment:
I think this method's name needs some adjustment to more accurately reflect what it should return when the bank data is not entered. Perhaps something similar to "bankIsoCountryCodeIsNotProvidedOrTranslatesToSingleFipsCountryCode" would be clearer.

Reply:
Adjusted name.